app/scripts/controllers/network/createInfuraClient.js

```diff
@@ -31,8 +42,10 @@ export default function createInfuraClient({ network, projectId }) {
     createRetryOnEmptyMiddleware({ blockTracker, provider: infuraProvider }),
     createBlockTrackerInspectorMiddleware({ blockTracker }),
     infuraMiddleware,
+    filterMiddleware,
```

Comment:
Oh no! I just realized, the order has changed here. These should be first in the list, but they're last. That probably explains the errors.

Reply:
Ah :)

Reply:
Made sure to put these first here: [469bbed](https://github.com/MetaMask/metamask-extension/pull/16693/commits/469bbed580be40d466cc76a97d7910f727eea5cb)